integration_testing/features/learner/learner-engage-content-recommended.feature

```diff
@@ -7,20 +7,20 @@ Feature: Learner engages recommended content
       And I am on *Learn > Recommended* page
 
   Scenario: Engage with content on Recommended tab
-    When I see content items under the *Most popular*, *Next steps* and *Resume* headings
+    When I see content items under the *Most popular* and *Recommended* headings
+    # *Next steps* and *Resume* ???
     Then I can use the right arrow buttons to see more in each section
       And after that I can use the left arrow buttons to browse back
     When I click the <content_item> content item
-    Then I see the *Recommended > '<content_item>'* breadcrumb
+    Then I see the *Recommended > '<content_item>'* page
       And I see the video start playing automatically
-      And below the player I see other *Recommended* items
+    #  And below the player I see other *Recommended* items
```

Comment:
@khangmach Please confirm that other recommended items below the currently viewed resource were removed intentionally in 0.13.

Reply:
Don't recall concrete decisions on this. Maybe @indirectlylit or @jtamiace can shed some light on this